sympy/printing/mathml.py

```diff
@@ -536,6 +536,13 @@ def _print_Relational(self, e):
         x.appendChild(self._print(e.rhs))
         return x
 
+    def _print_AppliedBinaryRelation(self, e):
```

Comment:
Since this is just a duplicate of the method above I think it would make more sense to have both methods call a helper.

Reply:
Thanks. I found out that this method had problem so I enhanced it and added tests.